### PR TITLE
Update phrasing and typos in `toolbarIcons.xml`

### PR DESCRIPTION
**Pull request by kurtmckee**
I saw this file after upgrading to Notepad++ 8.5, and wanted to contribute some phrasing fixes back to the project.

Thanks for your work on Notepad++!

_After creating the PR, I saw in the contributing doc that typo PR's a generally rejected. I'm submitting the changes in the hopes that there's a difference between typos that require translations to be updated, and typos that don't require translation updates. If not, then feel free to close._

### CODE REVIEW

**Comment by alankilborn** [comment]
Yes, typos should merely be pointed out in an "issue".  
Then a dev can do the actual fixing.
Thank you for spotting the typos.

**Comment by kurtmckee** [comment]
I'm surprised by this workflow style. It appears that the changes in this PR were accepted in their entirety (minus the end of file newline that GitHub introduced) in 399a8f302083facd49b3a78fc98e7646becb9be7.

Would it be possible to help me understand the goal of this workflow?

**Comment by alankilborn** [comment]
>Would it be possible to help me understand the goal of this workflow?

I could try, but I'm not sure what you're asking?

**Comment by kurtmckee** [comment]
I curious what goals are met by rejecting a typo PR by policy but then manually duplicating the changes into a commit?

For example, perhaps it's to reduce the number of contributors or copyright holders, or perhaps it's to make the git commits follow a certain shape.

I was surprised to see the PR rejected by policy, but the underlying changes were then copied wholesale into a different commit. I'm hoping to understand what goals were achieved by this workflow, because I haven't seen it before.

**Comment by alankilborn** [comment]
Oh.  Well, the typos that you located were adopted.  Wasn't that the goal?  :-)
Again, thank you for noticing the typos, and pointing them out.

If you would have pointed them out in an "issue" rather than a "PR", they would have been handled the same way as they were here.  A N++ dev would, if he agreed with your assessment of the typos as problems, made the changes himself.

If you mean, why wasn't your PR committed directly, I think the answer is that people who have done that in the past were then able to put on their CV:  "Notepad++ contributor", and, as far as github knows, it would corroborate that if anyone were to check.  Maybe there is more to it than that, but if so, I don't know the reasoning.

The bottom line is that by the contributing guidelines, such a PR should never exist in the first place.

**Comment by kurtmckee** [comment]
Got it. Thanks for your work on Notepad++!
